GliaWidgets/ViewController/Survey/Components/Survey.ValidationErrorView.swift

```diff
@@ -0,0 +1,35 @@
+import UIKit
+
+extension Survey {
+    final class ValidationErrorView: View {
+        let validationMessage = UILabel().make {
+            $0.numberOfLines = 0
+            $0.textColor = .red
+            $0.font = .boldSystemFont(ofSize: 12)
+            $0.text = "Please provide an answer."
```

Comment:
I suppose there will be a localization part. So in order to avoid missing localizations let's add some kind of `#warning` at least to indicated that and handle it in future.

Reply:
Yes, I'll move this string in the next PR together with theme stuff.